frontend/src/components/Editor/index.tsx

```diff
@@ -21,7 +21,8 @@ self.MonacoEnvironment = {
             default:
 				return new editorWorker();
 		}
-	}
+	},
+    createTrustedTypesPolicy: () => undefined
```

Comment:
```suggestion
	},
    // Can removed once https://github.com/microsoft/vscode/pull/185454 is released.
    createTrustedTypesPolicy: () => undefined,
```

See https://github.com/microsoft/vscode/pull/185454.